### PR TITLE
Fix opening SoundCloud links

### PR DESCRIPTION
**Pull request by wb9688**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

I got a NetworkOnMainThreadException though, so I just hacked `StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder().permitAll().build());` in, but we'll need to fix that. I know we're using RxJava for that, but I've no experience in it.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -55,7 +55,7 @@ dependencies {
         exclude module: 'support-annotations'
     }
 
-    implementation 'com.github.TeamNewPipe:NewPipeExtractor:1c97da8b51b3610'
+    implementation 'com.github.wb9688:NewPipeExtractor:6453805288'
```
Comment (TobiGr) [inline app/build.gradle]: I guess you didn't want to commit this 😆

Comment (wb9688) [reply app/build.gradle]: Yeah, but https://github.com/TeamNewPipe/NewPipeExtractor/pull/60 isn't merged yet.

**Comment by mauriciocolli** [comment]
If you can wait a bit, the network problem is fixed in #1018.

**Comment by theScrabi** [comment]
> If you can wait a bit, the network problem is fixed in #1018.
So do we still need this PR then?

**Comment by mauriciocolli** [comment]
@theScrabi this pull request modifies some things unreleated with mine, so yes.

**Comment by theScrabi** [comment]
All right.

File: app/src/main/java/org/schabi/newpipe/RouterActivity.java [hunk 2]
```diff
@@ -43,6 +44,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     protected void handleUrl(String url) {
+        StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder().permitAll().build()); // FIXME
         boolean success = NavigationHelper.openByLink(this, url);
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/RouterActivity.java]: I can not merge. The function `NaviagationHelper.openByLink()` seems not to bee exising in current dev branch.
@wb9688 could you please fix this merge collison.

Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/RouterActivity.java]: I can not merge. The function `NaviagationHelper.openByLink()` seems not to bee exising in current dev branch.
@wb9688 could you please fix this merge collison.

**Comment by theScrabi** [comment]
For the sake of cleaning the PR page, I'm going to close this for now. Please fix the abothe points and reopen this PR :)

**Pull request closed without merging**